connor/ge.go

```diff
@@ -11,8 +11,7 @@ import (
 // value is strictly larger than or equal to another.
 func ge(condition, data any) (bool, error) {
 	if condition == nil {
-		// Everything is greater than or equal to nil
-		return true, nil
+		return false, nil
```

Comment:
question: What caused this change?

Reply:
I dont think this change should be made in this PR.  Besides the lack of transparency in the change log, and cleaner PRs, it is also a good way of making sure that the `ge` change doesnt get merged for a few months.

Reply:
The rust implementation already behaves this way. I don't think it's a good idea to keep different behaviours. Otherwise you won't be able to switch between them without breaking existing tests.
And the idea behind it was anyway to get rid of golang version once we have Rust substitute because we don't want to maintain 2 implementations in parallel.

Reply:
> The rust implementation already behaves this way.

The rust implementation is not used.  Change the Go one, then the Rust one will pair up with it.